shared/AppInsightsCore/Tests/Selenium/ApplicationInsightsCore.Tests.ts

```diff
@@ -598,38 +597,25 @@ export class ApplicationInsightsCoreTests extends TestClass {
             test: () => {
                 let map = {};
 
-                let mwcRandCalled = 0;
-                let randCalled = 0;
+                // Check that mwcRandom is bing called (relies on the mwc implementation from the default seed)
+                mwcRandomSeed(1);
+                Assert.notEqual(722346555, random32(), "Make sure that the mwcRandom was not being called - step 1");
```

Comment:
Now also using "magic" numbers as mwc is psuedo random, so using the same seed always gives the same result (or should). So the test is also confirming that it really is using mwc.